getting_started/manage_your_plan/fuse_online.adoc

```diff
@@ -0,0 +1,22 @@
+[[getting-started-manage-your-fuse-plan]]
+= Manage Your Fuse Online Plan
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+[[customize-your-fuse-plan-with-add-ons]]
+== Customize Your Plan
+
+You can customize your plan with add-ons.
+
+. In the *Active Subscriptions* page, click the *Manage Subscription* link.
+
+. Click *Change* by the *Current Plan* heading
+
+. From the *Current Plan* page, click *Small*, *Medium*, or *Large*, then follow
```

Comment:
Just so you're aware, "small", "medium", and "large" are the terms we currently have but I don't know that they have been finalized 100%.

Reply:
@abhgupta When Fuse Online launches at the end of the month, will users be able to change plans yet, or is that coming at a later date? If so, I can just remove the Manage Your Fuse Online Plan page for now. Thanks!